app/config/FrontendAppConfig.scala

```diff
@@ -65,6 +65,8 @@ class FrontendAppConfig @Inject()(
   lazy val digitalcentre = loadConfig("accessibility-urls.digitalcentre")
   //accessibility Urls end
 
+  lazy val accountDashboardUrl = loadConfig("account_dashboard_statuses-urls.account_dashboard_statuses")
```

Comment:
This config doesn't seem to exist, can you please add it to `application.conf` or remove it if it's not needed?